OrbitQt/orbitmainwindow.ui

```diff
@@ -223,7 +223,7 @@
      <x>0</x>
      <y>0</y>
      <width>976</width>
-     <height>22</height>
```

Comment:
hmm. I'm not sure about this one.

Reply:
@beckerhe I think this should be the same as what happened to you when you cleaned up the UI files, right?